### ISSUE TITLE
Flow Router

### ISSUE THREAD

**Issue by zimt28** [yoolab/iron-router-i18n#47]
It would be cool if this would be just a `router-i18n` package and support flow-router as well :)

**Comment by martinopic** [comment]
I was already thinking about it ;-)

**Comment by zimt28** [comment]
That's cool :) I've already looked at flow router, and don't know how that could be done. But maybe you have an idea. My thoughts: https://github.com/meteorhacks/flow-router/issues/45 and https://github.com/meteorhacks/flow-router/issues/41

**Comment by martinopic** [comment]
I've had a first look, I think the same strategy (and even lots of code code) used for Iron Router i18n can be reused for Flowrouter, maybe this could be a reason more to also create a separate i18nConf package (see https://github.com/yoolab/iron-router-i18n/issues/46).

In any case it probably does not make much sense to have a generic "i18n router", better to have a separate package. I'll keep this issue open for updates.

**Comment by zimt28** [comment]
Okay, cool! Really interested in how you'll implement it :)

**Comment by laurentpayot** [comment]
Flow Router is now the ["first choice" router for MDG](http://meteor-guide.readthedocs.org/en/latest/routing/#installing-flow-router), but its lack of i18n prevents me from using it. Any news about Flow Router support? That would be really much appreciated!

**Comment by mwarren2** [comment]
I agree with @laurentpayot.  I'm a developer in europe, where multilanguage apps and sites are common. In my case they are the norm. But unfortunately i18n is usually an afterthought. I'm beginning to wonder if my first Meteor app will ever see the light of day. Just discovered also that both ms-seo for iron-router and flow-router-seo ignore 18n, for example. @martinopic your iron-router add-on was great, any chance of the same miracle for flow-router soon?

**Comment by laurentpayot** [comment]
I have just opened https://github.com/kadirahq/flow-router/issues/441 to see if there is hope or not...

**Comment by laurentpayot** [comment]
So there is no hope. Yet.

**Comment by tcastelli** [comment]
+1 for this idea

**Comment by marktomm** [comment]
+1 any hope yet?

**Comment by martinopic** [comment]
Hi, I might start working on a flow-router version on January but I cannot be sure at the moment. Sorry about that (I love flow-router too!)

**Comment by rafa-munoz** [comment]
+1 as well!

**Comment by laurentpayot** [comment]
@martinopic  Any news? I don't want to be pushy, iron-router-i18n is great and a flow-router version would be much appreciated by the community.

**Comment by WitaliB** [comment]
+1

**Comment by slissner** [comment]
+1 painfully needed as meteor 1.3 update crashed my app running on iron:router.
